dev-erlang/mqtree/mqtree-1.0.13.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
I mean, if we’re really going for this, how about EAPI 7?

Reply:
Since ejabberd bumps in gentoo tend to be rather large. I tried to keep the changes as minimal as possible. But yes, an EAPI 7 migration would be a sensible thing to do at some point.

Reply:
I get the point, although my thinking was it’s necessarily invasive anyway and we may as well get the EAPI 7 in unless there’s a reason _not_ to given we’re trying to push for that right now.

but I also understand it’s not your package, so maybe it’s better to do this separately. It just seems like a shame to add a bunch more EAPI 6 in one go if we can help it.